redex.py

```diff
@@ -579,7 +579,15 @@ def ensure_libs_dir(libs_dir, sub_dir):
         os.mkdir(libs_dir)
         os.mkdir(sub_dir)
         return libs_dir
-
+    
+def get_dex_file_path(args, extracted_apk_dir):
+    # base on file extension check if input is
+    # an apk file (".apk") or an Android bundle file (".aab")
+    log("APK: "+os.path.splitext(args.input_apk)[1])
+    if os.path.splitext(args.input_apk)[1] == ".aab":
```

Comment:
Deciding on the input is better. But we might want to assert (or maybe warn?) if the input and output types differ.

Reply:
I don't see any good reason why extension of the output file should be different from that of an input file. Why don't we assert it then?